tools/ddrgen/src/scanners/dwarf/AixDwarfParser.cpp

```diff
@@ -705,18 +279,15 @@ parseDeclarationLine (const string data, Parsed_Declaration_Data *parsedData)
 static int
 parseDwarfDie(const string line)
 {
-	
 	int ret = DW_DLV_OK;
 	/* get the type of the data of the line being parsed */
 	Parsed_Declaration_Data *parsedDeclarationData = new Parsed_Declaration_Data();
```

Comment:
The Parsed_Declaration_Data structure has strings in it, so I can't initialize it in that way without getting `std::logic_error: basic_string::_s_construct null not valid` since strings can't be initialized with the value 0.